### DISCUSSION TITLE
Alpine images

### DISCUSSION THREAD

**Discussion by ckulka** [ckulka/baikal-docker#55] [Ideas]
Another popular thing is providing Alpine-based images. This ticket will track my progress to provide one for Nginx and possibly Apache.

**Comment by BobWs** [comment]
Hi,
What's the status on this image? I'm interested in the Alpine-Apache version.
TIA

**Reply by ckulka** [reply]
Two years later and some light... while working on the GitHub Actions migration, I found out dependencies that were previously missing now exist in Alpine. If you'd be willing to test experimental builds for Apache and/or Nginx, I could look into building them.

**Reply by BobWs** [reply]
> Two years later and some light... while working on the GitHub Actions migration, I found out dependencies that were previously missing now exist in Alpine. If you'd be willing to test experimental builds for Apache and/or Nginx, I could look into building them.

Thanks but I’m already using the standard Apache version for a few years, so I’m quite happy with it. 
There is no reason for me to switch, so I guess for me there is no need to build it.
